velox/docs/develop/types.rst

```diff
@@ -134,10 +134,12 @@ Presto Type               Physical Type
 ========================  =====================
 HYPERLOGLOG               VARBINARY
 JSON                      VARCHAR
-TIMESTAMP WITH TIME ZONE  ROW<BIGINT, SMALLINT>
+TIMESTAMP WITH TIME ZONE  BIGINT
 ========================  =====================
 
 TIMESTAMP WITH TIME ZONE represents a time point in milliseconds precision
-from UNIX epoch with timezone information. Its physical type contains one 64-bit
-signed integer for milliseconds and another 16-bit signed integer for timezone ID.
-Valid range of timezone ID is [1, 1680], its definition can be found in ``TimeZoneDatabase.cpp``.
\ No newline at end of file
+from UNIX epoch with timezone information. Its physical type is BIGINT,
+The high 52 bits of bigint store signed integer for milliseconds, valid range of
```

Comment:
> valid range of milliseconds is [0xFFF8000000000000L, 0x7FFFFFFFFFFFF]

It is a bit hard to understand what the range is. Would it be possible to describe it using human-readable timestamps? E.g. something like "from 1970-01-01 12:30 to 2100-04-20 15:45".